### DISCUSSION TITLE
Load keys protected my module

### DISCUSSION THREAD

**Discussion by crisfint1425** [Keyfactor/ejbca-ce#575] [General]
Good morning
I change the protected keys from a hsm ncipher to another hsm, I'm trying to load the protected keys by module but when I try to start the ejbca service, In the web administration portal I only see the keys protected my ocs cards.
I have this version of ejbca EJBCA 6.3.1.1 Communitt and the new version of ncipher hsm is nshild connect xc

**Comment by primetomas** [comment]
EJBCA 6.3 is waaay old. Really recommend you to upgrade for security reasons, which includes upgrading many of your components to modern ones.
Y9u can find documentation about nShiled here: https://doc.primekey.com/ejbca/ejbca-integration/hardware-security-modules-hsm/ncipher-nshield-nethsm

**Reply by crisfint1425** [reply]
Hi Tomas, thank you, perhaps can you help us with support in order to updrade the ejbca version to the newest one please, and if it is possible, please tell us what the cost would be.

**Reply by primetomas** [reply]
You have all upgrade documentation here: https://doc.primekey.com/ejbca/ejbca-installation/upgrading-ejbca
If you need professional services assistance, don't hesitate to reach out to me and I can help with the right contacts.
